Packs/Impartner/Integrations/Impartner/Impartner.py

```diff
@@ -0,0 +1,161 @@
+
+    def get_accounts_list(self, params):
+        result = self._http_request(method="GET", url_suffix="/account", params=params, headers=self._headers)
+        return result
+
+    def get_accounts_id(self, id, params):
+        result = self._http_request(method="GET", url_suffix=f"/account/{id}", params=params,  headers=self._headers)
+        return result
+
+
+''' HELPER FUNCTIONS '''
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:  # pragma: no cover
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        res = client.get_accounts_list({})
+        if res.get('success'):
+            message = 'ok'
+    except DemistoException as e:
+        raise e
+    return message
+
+
+def impartner_get_account_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    query = args.get('query', None)
+    fields = args.get('fields', None)
+    filter = args.get('filter', None)
+    orderby = args.get('orderby', None)
+    skip = args.get('skip', None)
+    take = args.get('take', None)
+    all_fields = args.get('all_fields', None)
+    if all_fields == "TRUE":
+        fields = ""
+    params = assign_params(q=query, fields=fields, filter=filter, orderby=orderby, skip=skip, take=take)
+
+    # Call the Client function and get the raw response
+    result = client.get_accounts_list(params)
+    parsed_result = result.get('data', '')
+    readable_output = tableToMarkdown('List of account ID\'s', parsed_result.get('results'))
+    return CommandResults(
+        outputs_prefix='Impartner.Accounts.List',
+        readable_output=readable_output,
+        outputs_key_field='Impartner.Accounts.list.results.id',
+        outputs=parsed_result)
+
+
+def impartner_get_account_id_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    id = args.get('id', None)
+    fields = args.get('fields', None)
+    all_fields = args.get('all_fields', None)
+    if all_fields == "true":
+        fields = ""
+    params = assign_params(fields=fields)
+    # Call the Client function and get the raw response
+    result = client.get_accounts_id(id, params)
+    parsed_result = result.get('data', '')
+    readable_list = {'name': parsed_result['name'], 'ID': parsed_result['id'], 'link': parsed_result['recordLink'],
+                     'PST Engineer': parsed_result['tech_BD_Assigned_for_XSOAR__cf']}
+    readable_output = tableToMarkdown('Account Details', readable_list,
+                                      ['name', 'ID', 'link', 'PST Engineer'],
+                                      headerTransform=pascalToSpace, removeNull=False)
```

Comment:
Why removeNull=False?

Reply:
because pst_engineer can be empty - and we want to know when It's empty (Marcella didn't assign me or kobbi).
Ir removeNull is true, it won't show

Reply:
So you can remove the removeNull completely, by default its False